libraries/freertos_plus/standard/freertos_plus_tcp.cmake

```diff
@@ -0,0 +1,40 @@
+afr_module(NAME "freertos_plus_tcp" INTERNAL)
+
+set(src_dir "${CMAKE_CURRENT_LIST_DIR}/freertos_plus_tcp")
+set(inc_dir "${CMAKE_CURRENT_LIST_DIR}/freertos_plus_tcp/include")
+set(test_dir "${CMAKE_CURRENT_LIST_DIR}")
+
+# Create a list of all FReeRTOS+TCP headers files in the FreeRTOS+TCP
+# library.
+# The list of header files will be added to metadata required
+# for the FreeRTOS console.
+file(GLOB_RECURSE INCLUDE_HEADER_FILES "${inc_dir}/*.h")
```

Comment:
Do you need to do glob recurse or can it same as MQTT? https://github.com/aws/amazon-freertos/blob/master/libraries/core_mqtt_demo_dependencies.cmake#L12

Also how about more specific variable name FREERTOS_TCP_INCLUDE_HEADER_FILES?

Reply:
I replaced `GLOB_RECURSE` with just `GLOB`, and renamed the macro to `FREERTOS_TCP_HEADER_FILES`